src/assignments/tech-jobs-mvc.rst

```diff
@@ -3,4 +3,509 @@
+so you’ll focus on the controllers and views.
+
+In order to do this, you’ll be reusing the ``JobData`` class and
+``job_data.csv`` file from the console app. You will eventually have to go back
+and rewrite the data portion of the application to make a true, database-backed
+model. However, using the existing ``JobData`` class to provide some basic data
+functionality lets you focus on the views and controllers for now.
+
+Your Assignment
+---------------
+
+The list below provides a general overview of your assigned tasks. Specific
+details for each part appear in the following sections, so be sure to read them
+carefully as you solve each problem.
+
+#. Review Carly’s code in the ``JobData`` file as well as in the existing
+   controllers and views.
+#. As her next step, Carly created two basic templates to show lists of jobs
+   for each field of the data (``employer``, ``location``, etc.). You will need
+   to add some details to these views to allow users to select and display jobs
+   stored in the system.
+#. Finally, Carly started working on the search feature, but only got as far as
+   writing the code to display the search form. She’s handed the project to you
+   to finish the rest. You’ll add code to the controller and view to process
+   and display and search results.
+
+Getting Started
+----------------
+
+Set up a local copy of the project:
+
+#. Visit the `repository page <https://github.com/LaunchCodeEducation/java-web-dev-techjobs-mvc>`__
+   for this project. Fork the repository to create a copy under your own GitHub
+   account.
+#. Back in IntelliJ, close any open projects (after saving and committing your
+   work).
+#. From the welcome screen, click *Get From Version Control* and then select
+   the *Git* option.
+#. In the next panel, enter the URL for your fork, and then select the parent
+   directory where you want to store your project. Hit *Clone*.
+#. Launch the application (via the Gradle pane, *Tasks > Application >
+   bootRun*), so you can refer to both the code as well as the running app
+   as you complete the assignment.
+
+1) Review the Code
+-------------------
+
+.. admonition:: Tip
+
+   One essential programming skill that you will develop is the ability to read
+   and understand someone else's code. This assignment begins with you
+   practicing exactly that. Make sure you carefully examine the provided code
+   BEFORE you start changing things.
+
+   Trying to "fix" a code sample before understanding how it works leads to
+   confusion, frustration, and a broken program. DO NOT SKIP the code review!
+
+Carly created a Spring Boot application and filled in some features. She
+refactored ``JobData`` to generate an ArrayList of ``Job`` objects based on
+your TechJobs-OO work, and she added controllers and views for a Home, List and
+Search page. ``JobData`` now also builds ArrayLists for the ``Employer``,
+``Location``, ``PositionType``, and ``CoreCompetency`` objects.
+
+The Model
+^^^^^^^^^^
```

Comment:
One extra carrot!

Reply:
Removed.  Thanks.  (Does it really matter?)